sos/report/plugins/openstack_trove.py

```diff
@@ -51,13 +51,14 @@ def postproc(self):
         ]
         connection_keys = ["connection"]
 
+        join_con_keys = "|".join(connection_keys),
```

Comment:
Ditto the comma.

Reply:
too much time staring at the screen to try and figure things out